src/test/java/org/jenkinsci/plugins/webhookstep/WaitForWebhookTest.java

```diff
@@ -0,0 +1,117 @@
+package org.jenkinsci.plugins.webhookstep;
+
+import com.gargoylesoftware.htmlunit.WebClient;
+import hudson.FilePath;
+import hudson.model.Result;
+import jdk.nashorn.internal.parser.JSONParser;
+import net.sf.json.JSONObject;
+import net.sf.json.JSONSerializer;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.io.File;
+import java.net.URL;
+import java.util.List;
+
+import static net.javacrumbs.jsonunit.JsonAssert.assertJsonEquals;
+import static net.javacrumbs.jsonunit.JsonAssert.when;
+import static net.javacrumbs.jsonunit.core.Option.IGNORING_ARRAY_ORDER;
+import static org.junit.Assert.assertTrue;
+
+public class WaitForWebhookTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
```

Comment:
I thought you had something using `RestartableJenkinsRule`, or is that coming separately?